jest.config.ts.mjs

```diff
@@ -27,7 +27,7 @@ export default {
       modulePathIgnorePatterns: baseConfig.modulePathIgnorePatterns,
       roots: ['<rootDir>/packages'],
       runner: 'jest-runner-tsd',
-      testMatch: ['**/__typetests__/**/*.ts'],
+      testMatch: ['**/__typetests__/**/*.test.ts'],
```

Comment:
This was picking up `packages/jest-worker/__typetests__/testWorker.ts` and that is what it should do. Or?